README.md

```diff
@@ -1,3 +1,43 @@
 # Zed Nu
 
 This extension adds support for the [Nu](https://github.com/nushell/nushell) language.
+
+## Override Default Configuration (Optional)
+
+You can minimize the configuration autoloaded when the server starts,
+which may improve performance as complicated Nushell configurations can slow down the language server.
+
+```json
+{
+  "lsp": {
+    "nu": {
+      "binary": {
+        "command": "nu",
+        "arguments": ["--config", "~/.config/nushell/lsp.nu", "--lsp"]
+      }
+    }
+  }
+}
+```
+
+### Example of Minimal lsp.nu
+
+```nushell
+# Configure PATH to search for external command completions
+$env.PATH = $env.PATH
```

Comment:
This may be fine for now but once Zed is released on Windows using $env.PATH may be confusing since Windows typically uses $env.Path.

Reply:
Should we change it to `$env.Path` then, it seems that one works on macOS as well.

Reply:
If it's working well as case-insensitive, maybe it should just be `$env.path`. It's supposed to be case-insensitve, I just don't remember if there are bugs related to it.

Reply:
> If it's working well as case-insensitive, maybe it should just be `$env.path`. It's supposed to be case-insensitve, I just don't remember if there are bugs related to it.

Changed to `$env.path`. I don't worry about related bugs, as this is just an example. I think users know how to fixes PATH issues themselves, just as what's in their config.nu.

Reply:
> I think users know how to fixes PATH issues themselves, just as what's in their config.nu.

LOL. Believe me, that's a false statement for some. Let's run with these changes for now and see what happens.